docs/content/node-operation/past-sporks.mdx

```diff
@@ -5,11 +5,24 @@ description: Information about all Flow Mainnet and Testnet sporks.
 
 ## Mainnet Sporks
 
-## Mainnet 12
+## Mainnet 13
 
 | Spork Info                                                                              |
 | :-------------------------------------------------------------------------------------- |
 | **Access Node**: access.mainnet.nodes.onflow.org:9000                                   |
+| **Date**: Sept 15, 2021                                                                 |
+| **Root Height**: 18587478                                                               |
+| **Root Parent ID**: 2f708745fff4f66db88fac8f2f41d496edd341a2837d3e990e87679266e9bdb8    |
+| **Root State Commit**: 51e3098d327df22fd005d960cb73167c83cb438c53e6c4363c07d8611ae44528 |
+| **Git Commit**: 9535540110a4452231d044aceabab0e60f67708c                                |
+| **Branch/Tag**: [v0.21.3](https://github.com/onflow/flow-go/releases/tag/v0.21.3)       |
+| **Docker Tag**: v0.21.3                                                                 |
+
+## Mainnet 12
+
+| Spork Info                                                                              |
+| :-------------------------------------------------------------------------------------- |
+| **Access Node**: access-001.mainnet11.nodes.onflow.org:9000                             |
```

Comment:
is this correct?

Reply:
it is not - fixed. thanks for catching it